server_code/authorisation.py

```diff
@@ -7,9 +7,28 @@
 import functools
 
 import anvil.users
+from anvil.tables import app_tables
 
 __version__ = "2.6.0"
 
+config = {"get_roles_row": lambda user: anvil.users.get_user()["roles"]}
```

Comment:
seems unnecessary to call `get_user()` since we already have it

```suggestion
config = {"get_roles_row": lambda user: user"roles"]}
```